electrumx/server/env.py

```diff
@@ -43,6 +43,9 @@ def __init__(self, coin=None):
 
         # Core items
 
+        self.is_public = self.boolean('PUBLIC', False)
```

Comment:
IMO True would be better default. I expect most people will want True, and then it is one fewer thing to configure.